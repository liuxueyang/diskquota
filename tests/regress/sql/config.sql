--start_ignore
CREATE DATABASE diskquota;

\! gpconfig -c shared_preload_libraries -v diskquota 
\! gpstop -raf

\! gpconfig -c diskquota.naptime -v 0 
\! gpconfig -c max_worker_processes -v 20
\! gpstop -raf
--end_ignore

\c
-- Show the values of all GUC variables
SHOW diskquota.naptime;
SHOW diskquota.max_active_tables;
SHOW diskquota.worker_timeout;
