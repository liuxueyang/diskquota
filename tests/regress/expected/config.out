\c
-- Show the values of all GUC variables
SHOW diskquota.naptime;
 diskquota.naptime 
-------------------
 0
(1 row)

SHOW diskquota.max_active_tables;
 diskquota.max_active_tables 
-----------------------------
 1048576
(1 row)

SHOW diskquota.worker_timeout;
 diskquota.worker_timeout 
--------------------------
 60
(1 row)

